### ISSUE TITLE
Runtime error on OpenBSD 5.3 amd64 definition

### ISSUE THREAD

**Issue by wizardishungry** [mitchellh/veewee-to-packer#14]
```
    from /Users/jon/.rvm/rubies/ruby-1.9.2-p320/lib/ruby/1.9.1/fileutils.rb:1280:in `block (2 levels) in copy_file'
    from /Users/jon/.rvm/rubies/ruby-1.9.2-p320/lib/ruby/1.9.1/fileutils.rb:1279:in `open'
    from /Users/jon/.rvm/rubies/ruby-1.9.2-p320/lib/ruby/1.9.1/fileutils.rb:1279:in `block in copy_file'
    from /Users/jon/.rvm/rubies/ruby-1.9.2-p320/lib/ruby/1.9.1/fileutils.rb:1278:in `open'
    from /Users/jon/.rvm/rubies/ruby-1.9.2-p320/lib/ruby/1.9.1/fileutils.rb:1278:in `copy_file'
    from /Users/jon/.rvm/rubies/ruby-1.9.2-p320/lib/ruby/1.9.1/fileutils.rb:473:in `copy_file'
    from /Users/jon/.rvm/rubies/ruby-1.9.2-p320/lib/ruby/1.9.1/fileutils.rb:392:in `block in cp'
    from /Users/jon/.rvm/rubies/ruby-1.9.2-p320/lib/ruby/1.9.1/fileutils.rb:1423:in `block in fu_each_src_dest'
    from /Users/jon/.rvm/rubies/ruby-1.9.2-p320/lib/ruby/1.9.1/fileutils.rb:1439:in `fu_each_src_dest0'
    from /Users/jon/.rvm/rubies/ruby-1.9.2-p320/lib/ruby/1.9.1/fileutils.rb:1421:in `fu_each_src_dest'
    from /Users/jon/.rvm/rubies/ruby-1.9.2-p320/lib/ruby/1.9.1/fileutils.rb:391:in `cp'
    from /Users/jon/.rvm/gems/ruby-1.9.2-p320@veewee/gems/veewee-to-packer-0.2.3/lib/veewee-to-packer/builders/virtualbox.rb:146:in `block in convert'
    from /Users/jon/.rvm/gems/ruby-1.9.2-p320@veewee/gems/veewee-to-packer-0.2.3/lib/veewee-to-packer/builders/virtualbox.rb:143:in `each'
    from /Users/jon/.rvm/gems/ruby-1.9.2-p320@veewee/gems/veewee-to-packer-0.2.3/lib/veewee-to-packer/builders/virtualbox.rb:143:in `convert'
    from /Users/jon/.rvm/gems/ruby-1.9.2-p320@veewee/gems/veewee-to-packer-0.2.3/lib/veewee-to-packer.rb:116:in `block in convert'
    from /Users/jon/.rvm/gems/ruby-1.9.2-p320@veewee/gems/veewee-to-packer-0.2.3/lib/veewee-to-packer.rb:115:in `map'
    from /Users/jon/.rvm/gems/ruby-1.9.2-p320@veewee/gems/veewee-to-packer-0.2.3/lib/veewee-to-packer.rb:115:in `convert'
    from /Users/jon/.rvm/gems/ruby-1.9.2-p320@veewee/gems/veewee-to-packer-0.2.3/bin/veewee-to-packer:38:in `<top (required)>'
    from /Users/jon/.rvm/gems/ruby-1.9.2-p320@veewee/bin/veewee-to-packer:19:in `load'
    from /Users/jon/.rvm/gems/ruby-1.9.2-p320@veewee/bin/veewee-to-packer:19:in `<main>'
```

**Comment by mitchellh** [comment]
Fixed and released.

**Comment by viq** [comment]
I'm not sure it's resolved, I have gem 0.2.4 and when I try to convert https://github.com/jedi4ever/veewee/tree/master/templates/openbsd53_amd64 I get:

```
veewee-to-packer definition.rb 
An error occurred:

The kickstart file below is specified in the definition but
is a directory and not a file. The kickstart file list should be files.

/tmp/veewee/templates/openbsd53_amd64
```

**Comment by mitchellh** [comment]
OH! This happens when the kickstart file is an empty string! I'll fix this now.

**Comment by mitchellh** [comment]
Fixed, released 0.2.5

**Comment by viq** [comment]
Indeed, works, now, thank you!
